projects/plugins/jetpack/tests/e2e/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't think this file should be pushed. It should be in .gitignore.

Reply:
I can't believe I pushed that file. I'm definitely removing it.